.travis.yml

```diff
@@ -35,16 +33,17 @@ notifications:
   email:
       - blacklight-commits@googlegroups.com
```

Comment:
A bit unrelated to this PR, but are either of these notifications used? Maybe we should just delete them?

```suggestion
```

Reply:
will do!